packages/core/src/routes/verification-code.ts

```diff
@@ -0,0 +1,46 @@
+import { VerificationCodeType } from '@logto/connector-kit';
+import {
+  requestVerificationCodePayloadGuard,
+  verifyVerificationCodePayloadGuard,
+} from '@logto/schemas';
+
+import { createPasscode, sendPasscode, verifyPasscode } from '#src/libraries/passcode.js';
+import koaGuard from '#src/middleware/koa-guard.js';
+
+import type { AuthedRouter, RouterInitArgs } from './types.js';
+
+const codeType = VerificationCodeType.Generic;
+
+export default function verificationCodeRoutes<T extends AuthedRouter>(
+  ...[router]: RouterInitArgs<T>
+) {
+  router.post(
+    '/verification-code',
```

Comment:
Is there any possibility to make the route name as plural form?

Reply:
Why in plural form? You can only verify 1 code at a time

Reply:
updated to plural form